build-system/tasks/presubmit-checks.js

```diff
@@ -164,8 +164,8 @@ var forbiddenTerms = {
   'installStorageService': {
     message: privateServiceFactory,
     whitelist: [
-      'extensions/amp-analytics/0.1/amp-analytics.js',
-      'extensions/amp-analytics/0.1/storage-impl.js',
+      'src/runtime.js',
```

Comment:
I am actually not sure, it seems we do still list a lot of `installXService`. So I think this will stay here. Not sure why exactly we don't allow calling this.